src/FitbitProvider.php

```diff
@@ -83,12 +91,15 @@ protected function getDefaultScopes() {
      * @param  array|string $data Parsed response data
      * @return void
      */
-    protected function checkResponse(ResponseInterface $response, $data) {
-        if (static::DEBUG) {
-            error_log(json_encode($response));
-            error_log(json_encode($data));
-            error_log($response->getReasonPhrase());
-        }
+    protected function checkResponse(ResponseInterface $response, $data)
```

Comment:
No need to do an `if` here anymore. If the library user doesn't set a logger then the logs won't go anywhere. If they do set one it'll go to where they set the logger to log to.